charts/pega/charts/pegasearch/values.yaml

```diff
@@ -6,7 +6,8 @@ replicas: 1
 # node is set to the minimum number of master-eligible nodes that must be visible in order to form a cluster.
 # Configure this value using the formula (n/2) + 1 where n is replica count or desired capacity.
 minimumMasterNodes: 1
-# If externalURL is set, no search nodes will be deployed automatically, and Pega will use this search node url.
```

Comment:
lines 8 & 10: ...no search nodes are deployed automatically, and Pega uses this search node url